gnomad_qc/v4/create_release/create_release_sites_ht.py

```diff
@@ -224,8 +224,8 @@ def get_config(
             "custom_select": custom_info_select,
         },
         "freq": {
-            "ht": get_freq(data_type=data_type).ht(),
-            "path": get_freq(data_type=data_type).path,
+            "ht": get_freq(data_type=data_type, test=True).ht(),
```

Comment:
maybe you have to change this back

Reply:
Yeah I have a couple clean up things I need to do quick